### PR TITLE
ci: Added stale.yml for stale bot

### PR DESCRIPTION
**Pull request by ammarkarachi**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by yuth** [comment]
Is there a way for us to mark only issues with `pending-response` to go stale. Also we shold see if we have a way to remove pending-response as soon as customer responds in the issue

**Comment by ammarkarachi** [comment]
> Is there a way for us to mark only issues with `pending-response` to go stale. Also we shold see if we have a way to remove pending-response as soon as customer responds in the issue
I have put `pending-close-response-required` I can change it to `pending-response`

**Comment by brene** [comment]
@ammarkarachi - let's change it to pending responses because that's already the convention here.

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+daysUntilStale: 30
+daysUntilClose: 7
+exemptLabels:
+  - pinned
+  - security
+  - feature-request
+  - enhancement
+  - to-be-reproduced
+  - bug
+  - question
```
Comment (edwardfoyle) [inline .github/stale.yml:11]: should question be exempt? We have lots of questions that are pending-response and we never get more information. Seems like those should be auto-closed after a while

**Comment by brene** [comment]
> Also we shold see if we have a way to remove pending-response as soon as customer responds in the issue

@yuth - this is done automatically by the stale bot AFAIK. It's at least how it works in the JS repo.

**Pull request merged**